src/zenml/zen_stores/schemas/pipeline_run_schemas.py

```diff
@@ -599,6 +628,11 @@ def update(self, run_update: "PipelineRunUpdate") -> "PipelineRunSchema":
             The updated `PipelineRunSchema`.
         """
         if run_update.status:
+            self.status = run_update.status.value
```

Comment:
I think even this line here should be removed. The run status update should always happen via the SQLZenStore._update_run_status method anyway, no?

Reply:
I guess what I was trying to say is this: Even in SQLZenStore.update_run(...) we eventually call `_update_pipeline_run_status(...)`, which sets the status again. So why do we have to do it in the first place via the schema?